### DISCUSSION TITLE
How to use gatsby function other hosting platforms like apache with reverse proxy nginx?

### DISCUSSION THREAD

**Discussion by rajanmagar** [gatsbyjs/gatsby#34503] [Help]
I am looking for a solution to use gatsby function on nginx with reverse proxy environment. Gatsby documented only Gatsby Cloud support.

**Comment by violy** [comment]
Is your Apache host ready to run node.js scripts/functions ?

**Comment by rajanmagar** [comment]
How to do that? Can you share any source for reference?

On Fri, 28 Jan 2022 at 9:26 PM, Arthur Violy ***@***.***>
wrote:

> Is your Apache host ready to run node.js scripts/functions ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gatsbyjs/gatsby/discussions/34503#discussioncomment-2067489>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJOCDN5U36XHBMNUWOVSPH3UYK4EBANCNFSM5MCZ3TBQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
--------------------------------------------------------------------------------------------------------------

*"Do not wait to strike till the iron is hot, but make it hot by striking."*


Rajan Magar
Front End Dev
